Global frequency set at 1000000000000 ticks per second
src/mem/dram_interface.cc:690: warn: DRAM device capacity (8192 Mbytes) does not match the address range assigned (32 Mbytes)
src/base/statistics.hh:279: warn: One of the stats is a legacy stat. Legacy stat is a stat that does not belong to any statistics::Group. Legacy stat is deprecated.
board.remote_gdb: Listening for connections on port 7000
gem5 Simulator System.  https://www.gem5.org
gem5 is copyrighted software; use the --copyright option for details.

gem5 version 23.0.0.1--SSBSE Challeng Track
gem5 compiled Aug 13 2023 00:24:12
gem5 started Aug 22 2023 16:55:11
gem5 executing on fuzzer-03, pid 526703
command line: /home/ubuntu/gem5-ssbse-challenge-2023/build/X86/gem5.opt /home/ubuntu/gem5-ssbse-challenge-2023/ssbse-challenge-examples/hello-custom-binary-Ex.py --isa X86 --input /home/ubuntu/experiment-6/output-AUG-21/default/crashes//id:000006,sig:06,src:000106+000545,time:9399493,execs:12628,op:cm-gem5.so,pos:0

warn: No dot file generated. Please install pydot to generate the dot file and pdf.
src/sim/simulate.cc:194: info: Entering event queue @ 0.  Starting simulation...
src/sim/mem_state.cc:443: info: Increasing stack size by one page.
src/sim/syscall_emul.cc:74: warn: ignoring syscall mprotect(...)
src/sim/syscall_emul.cc:74: warn: ignoring syscall mprotect(...)
src/sim/syscall_emul.cc:74: warn: ignoring syscall mprotect(...)
Solution of Tower of Hanoi Problem with 0 Disks

Starting state:
A:  0  0  0  0 
B:  0  0  0  0 
C:  0  0  0  0 
------------------------------------------


Subsequent states:

A:  0  0  0  0 
B:  0  0  0  0 
C:  0  0  0  0 
------------------------------------------
A:  0  0  0  0 
B:  0  0  0  0 
C:  0  0  0  0 
------------------------------------------
A:  0  0  0  0 
B:  0  0  0  0 
C:  0  0  0  0 
------------------------------------------
A:  0  0  0  0 
B:  0  0  0  0 
C:  0  0  0  0 
------------------------------------------
A:  0  0  0  0 
B:  0  0  0  0 
C:  0  0  0  0 
------------------------------------------
A:  0  0  0  0 
B:  0  0  0  0 
C:  0  0  0  0 
------------------------------------------
A:  0  0  0  0 
B:  0  0  0  0 
C:  0  0  0  0 
------------------------------------------
A:  0  0  0  0 
B:  0  0  0  0 
C:  0  0  0  0 
------------------------------------------
A:  0  0  0  0 
B:  0  0  0  0 
C:  0  0  0  0 
------------------------------------------
A:  0  0  0  0 
B:  0  0  0  0 
C:  0  0  0  0 
------------------------------------------
A:  0  0  0  0 
B:  0  0  0  0 
C:  0  0  0  0 
------------------------------------------
A:  0  0  0  0 
B:  0  0  0  0 
C:  0  0  0  0 
------------------------------------------
A:  0  0  0  0 
B:  0  0  0  0 
C:  0  0  0  0 
------------------------------------------
A:  0  0  0  0 
B:  0  0  0  0 
C:  0  0  0  0 
------------------------------------------
A:  0  0  0  0 
B:  0  0  0  0 
C:  0  0  0  0 
------------------------------------------
src/arch/x86/faults.cc:131: panic: Unrecognized/invalid instruction executed:
 
{
	leg = 0,
	rex = 0,
	vex/xop = 0,
	op = {
		type = one byte,
		op = 0xff,
		},
	modRM = 0xff,
	sib = 0,
	immediate = 0,
	displacement = 0
	dispSize = 0}
Memory Usage: 237652 KBytes
Program aborted at tick 23471937567
--- BEGIN LIBC BACKTRACE ---
/home/ubuntu/gem5-ssbse-challenge-2023/build/X86/gem5.opt(+0x38b2663)[0x5593be055663]
/home/ubuntu/gem5-ssbse-challenge-2023/build/X86/gem5.opt(+0x39060c9)[0x5593be0a90c9]
/lib/x86_64-linux-gnu/libpthread.so.0(+0x14420)[0x7f91bb7bc420]
/lib/x86_64-linux-gnu/libc.so.6(gsignal+0xcb)[0x7f91ba8f200b]
/lib/x86_64-linux-gnu/libc.so.6(abort+0x12b)[0x7f91ba8d1859]
/home/ubuntu/gem5-ssbse-challenge-2023/build/X86/gem5.opt(+0xab5f45)[0x5593bb258f45]
/home/ubuntu/gem5-ssbse-challenge-2023/build/X86/gem5.opt(+0x28978fa)[0x5593bd03a8fa]
/home/ubuntu/gem5-ssbse-challenge-2023/build/X86/gem5.opt(+0x1f6f583)[0x5593bc712583]
/home/ubuntu/gem5-ssbse-challenge-2023/build/X86/gem5.opt(+0x1f55285)[0x5593bc6f8285]
/home/ubuntu/gem5-ssbse-challenge-2023/build/X86/gem5.opt(+0x1f534dd)[0x5593bc6f64dd]
/home/ubuntu/gem5-ssbse-challenge-2023/build/X86/gem5.opt(+0x38e3556)[0x5593be086556]
/home/ubuntu/gem5-ssbse-challenge-2023/build/X86/gem5.opt(+0x395010d)[0x5593be0f310d]
/home/ubuntu/gem5-ssbse-challenge-2023/build/X86/gem5.opt(+0x3951e19)[0x5593be0f4e19]
/home/ubuntu/gem5-ssbse-challenge-2023/build/X86/gem5.opt(+0x17f771b)[0x5593bbf9a71b]
/home/ubuntu/gem5-ssbse-challenge-2023/build/X86/gem5.opt(+0xa626b9)[0x5593bb2056b9]
/lib/x86_64-linux-gnu/libpython3.8.so.1.0(+0x2a8748)[0x7f91bba75748]
/lib/x86_64-linux-gnu/libpython3.8.so.1.0(_PyEval_EvalFrameDefault+0x8dd8)[0x7f91bb84af48]
/lib/x86_64-linux-gnu/libpython3.8.so.1.0(_PyEval_EvalCodeWithName+0x8fb)[0x7f91bb997e4b]
/lib/x86_64-linux-gnu/libpython3.8.so.1.0(_PyFunction_Vectorcall+0x94)[0x7f91bba75124]
/lib/x86_64-linux-gnu/libpython3.8.so.1.0(+0x74d6d)[0x7f91bb841d6d]
/lib/x86_64-linux-gnu/libpython3.8.so.1.0(_PyEval_EvalFrameDefault+0x7d86)[0x7f91bb849ef6]
/lib/x86_64-linux-gnu/libpython3.8.so.1.0(_PyEval_EvalCodeWithName+0x8fb)[0x7f91bb997e4b]
/lib/x86_64-linux-gnu/libpython3.8.so.1.0(_PyFunction_Vectorcall+0x94)[0x7f91bba75124]
/lib/x86_64-linux-gnu/libpython3.8.so.1.0(+0x74d6d)[0x7f91bb841d6d]
/lib/x86_64-linux-gnu/libpython3.8.so.1.0(_PyEval_EvalFrameDefault+0xea8)[0x7f91bb843018]
/lib/x86_64-linux-gnu/libpython3.8.so.1.0(_PyEval_EvalCodeWithName+0x8fb)[0x7f91bb997e4b]
/lib/x86_64-linux-gnu/libpython3.8.so.1.0(PyEval_EvalCodeEx+0x42)[0x7f91bb9981d2]
/lib/x86_64-linux-gnu/libpython3.8.so.1.0(PyEval_EvalCode+0x1f)[0x7f91bb9985bf]
/lib/x86_64-linux-gnu/libpython3.8.so.1.0(+0x1cfc01)[0x7f91bb99cc01]
/lib/x86_64-linux-gnu/libpython3.8.so.1.0(+0x25f547)[0x7f91bba2c547]
/lib/x86_64-linux-gnu/libpython3.8.so.1.0(+0x74d6d)[0x7f91bb841d6d]
/lib/x86_64-linux-gnu/libpython3.8.so.1.0(_PyEval_EvalFrameDefault+0x12fd)[0x7f91bb84346d]
--- END LIBC BACKTRACE ---
For more info on how to address this issue, please visit https://www.gem5.org/documentation/general_docs/common-errors/ 

./mine-crash.sh: line 21: 526703 Aborted                 $gem5 $gem5_script --isa X86 --input $txt_file
